frontend/packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/pipeline-details/PipelineVisualizationStepList.tsx

```diff
@@ -48,30 +49,39 @@ export const PipelineVisualizationStepList: React.FC<PipelineVisualizationStepLi
   isSpecOverview,
   taskName,
   steps,
-}) => (
-  <div className="odc-pipeline-vis-steps-list">
-    <div className="odc-pipeline-vis-steps-list__task-name">{taskName}</div>
-    {steps.map(({ duration, name, runStatus: status }) => {
-      return (
-        <div
-          className={classNames('odc-pipeline-vis-steps-list__step', {
-            'odc-pipeline-vis-steps-list__step--task-run': !isSpecOverview,
-          })}
-          key={name}
-        >
-          {!isSpecOverview ? (
-            <div className="odc-pipeline-vis-steps-list__icon">
-              <TooltipColoredStatusIcon status={status} />
-            </div>
-          ) : (
-            <span className="odc-pipeline-vis-steps-list__bullet">&bull;</span>
-          )}
-          <div className="odc-pipeline-vis-steps-list__name">{name}</div>
-          {!isSpecOverview && (
-            <div className="odc-pipeline-vis-steps-list__duration">{duration}</div>
-          )}
+  isFinallyTask,
+}) => {
+  const { t } = useTranslation();
+  return (
+    <div className="odc-pipeline-vis-steps-list">
```

Comment:
I know this is how it was written before, but the className should match the file name
```suggestion
    <div className="odc-pipeline-visualization-step-list">
```

Reply:
done